deployment-aws/terraform/storage.tf

```diff
@@ -41,3 +45,94 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "cromwell_storage_
     bucket_key_enabled = true
   }
 }
+
+resource "aws_s3_bucket_policy" "logging_bucket_policy"  {
+  bucket = aws_s3_bucket.log_bucket.id
+  policy = data.aws_iam_policy_document.logging_bucket_policy_doc.json
+}
+
+# For specifying the policies of the log bucket, we use an `aws_iam_policy_document`
+# resource rather than using HEREDOC syntax or otherwise. See:
```

Comment:
Comments should explain why was something done instead of what was done and here an explanation wouldn't be necessary if an inline policy was used